fili-generic-example/src/main/java/com/yahoo/wiki/webservice/data/config/dimension/GenericDimensionField.java

```diff
@@ -0,0 +1,42 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.wiki.webservice.data.config.dimension;
+
+import com.yahoo.bard.webservice.data.dimension.DimensionField;
+import com.yahoo.bard.webservice.util.EnumUtils;
+
+/**
+ * DimensionField enum.
+ */
+public enum GenericDimensionField implements DimensionField {
+    ID("Dimension ID"),
+    DESC("Dimension Description");
+
```

Comment:
I'm not sure 'DESC' is good for anything on dimensions loaded directly from druid.